pallets/staking/src/lib.rs

```diff
@@ -2102,7 +2102,7 @@ pub mod pallet {
             let controller = ensure_signed(origin)?;
             let mut ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
             T::Currency::remove_lock(STAKING_ID, &ledger.stash);
-            let amount = ledger.slash(amount, T::Currency::minimum_balance());
+            let amount = ledger.slash(amount, <ExistentialDeposit<T>>::get());
```

Comment:
This method is to burn a validator's bond, we should allow to burn all the bond(only keep 0.2 DPR to keep account alive), so no need to change here.